### ISSUE TITLE
Standard template for docs/apps 

### ISSUE THREAD

**Issue by yeqbfgxjiq** [1Hive/1Hive#15]
TL;DR: a standard template for pages in the apps section of the docs.

As we develop our own apps like [Redemptions](https://github.com/1Hive/redemptions), we're going to have to build and maintain our own docs, guides, and tutorials.
- @0xGabi had a great idea to use [the scripts in hack.aragon to link GitHub comments to the docs](https://github.com/aragon/hack/tree/master/website/scripts). I think that makes a lot of sense, creating a single source of truth (GitHub) and reducing the amount of lines we have to write/maintain (DRY).
- As @lkngtn mentioned it would also be awesome if there was a table with basic info like who the maintainer is, where the repo is, and whether each app is available on rinkeby only or on mainnet.

**Comment by yeqbfgxjiq** [comment]
This would be great to develop for the Redemptions App as a showcase of what 1Hive wants to create for the Nest Grant apps. 
- @0xGabi Can you please explain a little more how the hack.aragon scripts work and what they do?
- @lkngtn Did you have more ideas on what a standard docs template might look like for 1Hive?

**Comment by 0xGabi** [comment]
@burrrata The script will fetch the docs on each repo to get the last version and build the Docusaurus markdown files. I can do it for the Redemptions app as a first example. 

About what a doc template might look like...for each app we should include this topics at least:
1. Intro & run locally (with the provide template). Maybe this could even be the Readme of each app.
2. Deployment & install on existing dao.

**Comment by yeqbfgxjiq** [comment]
@0xGabi That sounds awesome! <s>Let me know if I can help in any way :)</s> Let me know when you start working on the repo/docusaurus docs so that I can learn how that works and build it into the app docs kit/template we'll ship along with each app.

**Comment by yeqbfgxjiq** [comment]
Now that we're working on this as part of the [1Hive DAO Kit](https://github.com/1Hive/1hive-dao-kit) are we good to close this issue?

**Comment by yeqbfgxjiq** [comment]
Closing this because the [1Hive DAO Kit #2](https://github.com/1Hive/1hive-dao-kit/issues/2) is where this is being developed. Moved everything from here over there, so that is now the latest version with up to date info.
